config/initializers/wrap_parameters.rb

```diff
@@ -7,9 +7,9 @@
 
 # Enable parameter wrapping for JSON. You can disable this by
 # setting :format to an empty array.
-ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
-end
+# ActiveSupport.on_load(:action_controller) do
+#   wrap_parameters format: [:json]
+# end
```

Comment:
Was getting `Unpermitted parameter: :graphql` from graphql_controller. Since not building out any other API endpoints, no need for wrappind

Reply:
If you add `:graphql` to the permitted strong params in that controller, does that help?

Reply:
IMO it's cleaner/logical to delete code rather than leave commented